src/hooks/useAppBootstrap.js

```diff
@@ -50,6 +57,18 @@ export const useAppBootstrap = client => {
             route: routes.welcome
           })
         }
+      } else if (OnboardingRedirection) {
+        const onboardingRedirectionURL = formatOnboardingRedirection(
+          OnboardingRedirection,
+          client
+        )
+
+        return setInitialRoute({
```

Comment:
Maybe we should clear `OnboardingRedirection` or it will be used if the user logs out and tries to login into another Cozy.

Reply:
Done